Patches/YAMLcpp/CMakeLists.txt

```diff
@@ -311,12 +311,18 @@ export(
 export(PACKAGE yaml-cpp)
 set(EXPORT_TARGETS yaml-cpp CACHE INTERNAL "export targets")
 
-set(CONFIG_INCLUDE_DIRS "${YAML_CPP_SOURCE_DIR}/include")
+set(CONFIG_INCLUDE_DIRS "${INCLUDE_INSTALL_DIR}")
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/yaml-cpp-config.cmake.in
 	"${PROJECT_BINARY_DIR}/yaml-cpp-config.cmake" @ONLY)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/yaml-cpp-config-version.cmake.in
 	"${PROJECT_BINARY_DIR}/yaml-cpp-config-version.cmake" @ONLY)
 
+set(CMAKE_CONFIG_FILES
+       ${PROJECT_BINARY_DIR}/yaml-cpp-config.cmake
+       ${PROJECT_BINARY_DIR}/yaml-cpp-config-version.cmake
+       ${PROJECT_BINARY_DIR}/yaml-cpp-targets.cmake)
+install(FILES ${CMAKE_CONFIG_FILES} DESTINATION ${CMAKE_INSTALL_PREFIX}/cmake)
```

Comment:
Here cmake is lowercase, but I think in the External_YAMLCPP.cmake you have CMake. I prefer the lowercase version.

Reply:
I prefer the lower case too, but there are lots of stuff with CMake (like Ceres and Darknet), and the linux box I checked had fletch/install/CMake, I think I need to make it 'CMake'. Then we can go and look for consistency and put up a different PR with all packages changed to cmake

I think... dumb ol case sensitivities ... since when did operating systems get so pc (pun intended)